app/controller.js

```diff
@@ -4,12 +4,31 @@ module.exports = store => {
     return store.saveUser(user);
   };
 
+
   const findAllUsers = Users => {
     return store.findAllUsers(Users)
   }
 
+
```

Comment:
I think in this case you don't have to wrap an existing method in another function with the same name. I think in this case you can directly use the store wherever you need injecting them as you did in this module. Are you trying to use the proxy pattern here for some reason?  The proxy pattern can make sense if you want to protect the store by validating or transforming things in the functions you're wrapping...

Reply:
@servatj you're totally correct. However this was done so @ScottRhodie can get his head around the concept of a controller as proxy that could do more than that